aws_auth.tf

```diff
@@ -38,11 +37,27 @@ locals {
     }
   ]
 
+  auth_launch_template_worker_roles_ext = [
+    for k, v in local.worker_group_launch_template_configurations_ext : {
+      worker_role_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/${var.manage_worker_iam_resources ? aws_iam_instance_profile.workers_launch_template_ext[k].role : data.aws_iam_instance_profile.custom_worker_group_launch_template_iam_instance_profile_ext[k].role_name}"
```

Comment:
Need to replace the `:aws:` portion of the ARN with `:${data.aws_partition.current.partition}:` (same in auth_worker_roles_ext below)

Reply:
fixed